fineract-provider/src/main/java/org/apache/fineract/accounting/glaccount/domain/GLAccount.java

```diff
@@ -77,10 +82,6 @@ public class GLAccount extends AbstractPersistableCustom {
     @JoinColumn(name = "tag_id")
     private CodeValue tagId;
 
-    protected GLAccount() {
-        //
-    }
-
     private GLAccount(final GLAccount parent, final String name, final String glCode, final boolean disabled,
             final boolean manualEntriesAllowed, final Integer type, final Integer usage, final String description, final CodeValue tagId) {
         this.name = StringUtils.defaultIfEmpty(name, null);
```

Comment:
This is also one of those nonsensical constructs... we don't need this... please remove... also: private constructor... not ideal for a JPA entity... for JPA entity classes let's have:
- one public no args constructor
- one public all args constructor (if necessary... that means if we had one in place before with any scope private, protected or public); in an ideal world I would see only the no args constructor on entity classes